otx/algorithms/detection/utils/utils.py

```diff
@@ -112,46 +111,6 @@ def load_template(path):
     return template
 
 
-@check_input_parameters_type()
-def get_task_class(path: str):
-    """Return Task classes."""
-    module_name, class_name = path.rsplit(".", 1)
-    module = importlib.import_module(module_name)
-    return getattr(module, class_name)
-
-
-class TrainingProgressCallback(TimeMonitorCallback):
-    """TrainingProgressCallback class for time monitoring."""
-
-    def __init__(self, update_progress_callback):
-        super().__init__(0, 0, 0, 0, update_progress_callback=update_progress_callback)
-
-    def on_train_batch_end(self, batch, logs=None):
-        """Callback function on training batch ended."""
-        super().on_train_batch_end(batch, logs)
-        self.update_progress_callback(self.get_progress())
-
-    def on_epoch_end(self, epoch, logs=None):
-        """Callback function on epoch ended."""
-        self.past_epoch_duration.append(time.time() - self.start_epoch_time)
-        self._calculate_average_epoch()
-        score = None
-        if hasattr(self.update_progress_callback, "metric") and isinstance(logs, dict):
-            score = logs.get(self.update_progress_callback.metric, None)
-            score = float(score) if score is not None else None
-            if score is not None:
-                iter_num = logs.get("current_iters", None)
-                if iter_num is not None:
-                    print(f"score = {score} at epoch {epoch} / {int(iter_num)}")
-                    # as a trick, score (at least if it's accuracy not the loss) and iteration number
-                    # could be assembled just using summation and then disassembeled.
-                    if score < 1.0:
-                        score = score + int(iter_num)
-                    else:
-                        score = -(score + int(iter_num))
-        self.update_progress_callback(self.get_progress(), score=score)
-
-
 class InferenceProgressCallback(TimeMonitorCallback):
```

Comment:
This will be moved to `common`. It will be held in a[ segmentation PR](https://github.com/openvinotoolkit/training_extensions/pull/1318), not this PR.